guidelines/index.html

```diff
@@ -1586,23 +1586,23 @@ <h1 id="title" class="title">Web Content Accessibility Guidelines (WCAG) 2.2</h1
     <ul>
-        <li><strong>Spacing:</strong> Undersized targets (those less than 24 by 24 CSS pixels) are positioned so that if a 24 CSS pixel diameter circle is centered on the <a href="#dfn-bounding-boxes" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-bounding-boxes-1" title="[New]">bounding box</a> of each, the circles do not intersect another target or the circle for another undersized target;</li>
-        <li><strong>Equivalent:</strong> The function can be achieved through a different control on the same page that meets this criterion;</li>
-        <li><strong>Inline:</strong> The target is in a sentence or its size is otherwise constrained by the line-height of non-target text;</li>
-        <li><strong>User agent control:</strong> The size of the target is determined by the <a href="#dfn-user-agents" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-user-agents-3" title="ウェブコンテンツを取得して利用者に提示するあらゆるソフトウェア。">user agent</a> and is not modified by the author;</li>
-        <li><strong>Essential:</strong> A particular <a href="#dfn-presentation" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-presentation-4" title="rendering of the content in a form to be perceived by users">presentation</a> of the target is <a href="#dfn-essential" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-essential-15" title="もし取り除いてしまうと、コンテンツの情報又は機能を根本的に変えてしまい、かつ、適合する他の方法では情報及び機能を実現できない。">essential</a> or is legally required for the information being conveyed.</li>
+        <li><strong>スペースがある:</strong> 小サイズ (24 × 24 CSS ピクセル未満) のターゲットにおいて、各ターゲットの<a href="#dfn-bounding-boxes" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-bounding-boxes-1" title="[New]">バウンディングボックス</a>の中心に直径 24 CSS ピクセルの円を置いた場合、その円が別のターゲットもしくは別の小サイズのターゲットの円と交差しないように、ターゲットが配置されている。</li>
+        <li><strong>同等のものが存在する:</strong> その機能を、同一ページ上の、この達成基準を満たす別のコントロールによって、実行することができる。</li>
+        <li><strong>インラインである:</strong> そのターゲットが文中に存在する、又は、そのサイズがターゲットでないテキストの行送りによって制約される。</li>
+        <li><strong>ユーザエージェントのコントロールである:</strong> ターゲットのサイズが<a href="#dfn-user-agents" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-user-agents-3" title="ウェブコンテンツを取得して利用者に提示するあらゆるソフトウェア。">ユーザエージェント</a>によって定められており、かつコンテンツ制作者が変更していない。</li>
+        <li><strong>必要不可欠:</strong> そのターゲットを特定の方法で<a href="#dfn-presentation" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-presentation-4" title="利用者が知覚できる形式でコンテンツをレンダリングすること。">提示</a>することが、情報伝達にとって<a href="#dfn-essential" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-essential-15" title="もし取り除いてしまうと、コンテンツの情報又は機能を根本的に変えてしまい、かつ、適合する他の方法では情報及び機能を実現できない。">必要不可欠</a>である、又は法的に要求されている。</li>
```

Comment:
ちょっと自信がないですが、required forでかたまりになっているのでは、と思いましたのでそのようにしてみました。

```suggestion
        <li><strong>必要不可欠:</strong> そのターゲットを特定の方法で<a href="#dfn-presentation" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-presentation-4" title="利用者が知覚できる形式でコンテンツをレンダリングすること。">提示</a>することが<a href="#dfn-essential" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-essential-15" title="もし取り除いてしまうと、コンテンツの情報又は機能を根本的に変えてしまい、かつ、適合する他の方法では情報及び機能を実現できない。">必要不可欠</a>である、又は伝達される情報に法律上要求されている。</li>
```

Reply:
達成基準 2.5.5 の訳が

> A particular presentation of the target is essential to the information being conveyed.
> そのターゲットを特定の方法で提示することが、**情報伝達にとって必要不可欠である**。

...となっているので、それに呼応する形で、 2.5.8 の訳でも

> そのターゲットを特定の方法で提示することが、**情報伝達にとって必要不可欠である**、又は法的に要求されている。

...としました。

「for the information being conveyed.」が「is legally required」と「is essential」の両方にかかっている、という理解です。

Reply:
@bakera さんのおっしゃるとおり、たしかに私の案だと、「情報伝達にとって」が「必要不可欠」だけにかかっているように読める余地がありますね。

達成基準 2.5.5 と合わせて、以下のように修正する、というのはいかがでしょうか？(既存の 2.5.5 の訳で「情報伝達」と訳されているところを、今回の 2.5.8 の訳に合わせて、information being conveyed という原文に忠実に「伝達される情報」としています。)

達成基準 2.5.5 の Essential

> Essential: A particular presentation of the target is essential to the information being conveyed.
> 必要不可欠 : そのターゲットを特定の方法で提示することが、伝達される情報において必要不可欠である。

達成基準 2.5.8 の Essential

> Essential: A particular presentation of the target is essential or is legally required for the information being conveyed.
> 必要不可欠 : そのターゲットを特定の方法で提示することが、必要不可欠である、又は伝達される情報に対して法的に要求されている。

Reply:
良いと思います!

Reply:
ありがとうございます！修正しました。[5b93e30](https://github.com/waic/wcag22/pull/38/commits/5b93e30e14209e907f2fc2892a2515112023e8f9)